### PR TITLE
feat: update Talk user manual screenshots for v30

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Fix screenshots (only) for 30 release

### 🖼️ Screenshots

See [html build.zip](https://github.com/user-attachments/files/16885703/html.build.zip)

### CODE REVIEW

File: user_manual/talk/images/new-room.png
Comment (DorraJaouad) [inline user_manual/talk/images/new-room.png:1]: I would expect to see a room called "Company" just created and first message, following the above screenshots

File: user_manual/talk/images/clear-filter.png
Comment (DorraJaouad) [inline user_manual/talk/images/clear-filter.png:1]: Not critical but hover should be on "clear filters" :p

**Review by DorraJaouad — APPROVED** [review]
Looks great overall

**Review by ShGKme — APPROVED** [review]

**Review by DorraJaouad — APPROVED** [review]
Good to go, nice work!

**Pull request merged**